### ISSUE TITLE
Learning rate is not changed in accordance with the scheduler after resuming [FasterRCNN example]

### ISSUE THREAD

**Issue by chunfuchen** [ppwwyyxx/tensorpack#535]
Bug Reports: PLEASE always include
1. What you did. (command you run if using examples; post or describe your code if not)

Resume a model from a checkpoint.

2. What you observed, e.g. logs.

The learning rate is not changed according to the scheduler.

3. What you expected, if not obvious.

Learning rate is scheduled to change from 0.01 to 0.001 at epoch 500; however, after 500 epoch, the learning rate is still 0.01.

4. Your environment (TF version, cudnn version, number & type of GPUs), if it matters.

TF 1.4.0, cudnn 7.0.3, 8 v100 GPUs., Tensorpack version 0.8.0

This is related to the examples of FasterRCNN, in my case, `warmup_epoch= 5`, `factor=1`, `st3pnum=300`. 

I resume the model at `global_step=147000`. According to my scheduler, the learning rate should change to 0.001 after `global_step=150000` a.k.a. `epoch=500`

```
callbacks=[
				PeriodicTrigger(ModelSaver(), every_k_epochs=5),
				ScheduledHyperParamSetter(
					'learning_rate',
					[(warmup_epoch * factor, 1e-2),
					 (150000 * factor // stepnum, 1e-3),
					 (230000 * factor // stepnum, 1e-4)]),
				EvalCallback(),
				GPUUtilizationTracker(),
			]
```

**Comment by chunfuchen** [comment]
Please ignore it now... something wrong from my side....

**Comment by chunfuchen** [comment]
okay, my bad. but the start epoch is read from `stat.json` rather than deriving from `global_step` and `step_num`
